src/server/sql/reading/create_reading_views.sql

```diff
@@ -63,293 +65,243 @@ END;
+				(r.reading * 3600 / u.sec_in_rate) *
+				-- The number of seconds that the reading shares with the interval
+				EXTRACT(EPOCH FROM LEAST(r.end_timestamp, gen.interval_start + INTERVAL '1 hour') - GREATEST(r.start_timestamp, gen.interval_start))
+			) /
+			-- The number of seconds that the reading shares with the interval
+			EXTRACT(EPOCH FROM LEAST(r.end_timestamp, gen.interval_start + INTERVAL '1 hour') - GREATEST(r.start_timestamp, gen.interval_start))
+			)
+		END AS max_rate,
+
+		CASE
+		WHEN u.unit_represent = 'quantity'::unit_represent_type THEN
+			MIN(
+			--Extract the minimum rate over each day
+			(
+				-- Reading rate in kw
+				(r.reading * 3600 / EXTRACT(EPOCH FROM (r.end_timestamp - r.start_timestamp))) *
+				-- The number of seconds that the reading shares with the interval
+				EXTRACT(EPOCH FROM LEAST(r.end_timestamp, gen.interval_start + INTERVAL '1 hour') - GREATEST(r.start_timestamp, gen.interval_start))
+			) /
+			-- The number of seconds that the reading shares with the interval
+			EXTRACT(EPOCH FROM LEAST(r.end_timestamp, gen.interval_start + INTERVAL '1 hour') - GREATEST(r.start_timestamp, gen.interval_start))
+			)
+		WHEN u.unit_represent IN ('flow'::unit_represent_type, 'raw'::unit_represent_type) THEN
+			MIN(
+			(
+				-- Reading rate in kw
+				(r.reading * 3600 / u.sec_in_rate) *
+				-- The number of seconds that the reading shares with the interval
+				EXTRACT(EPOCH FROM LEAST(r.end_timestamp, gen.interval_start + INTERVAL '1 hour') - GREATEST(r.start_timestamp, gen.interval_start))
+			) /
+			-- The number of seconds that the reading shares with the interval
+			EXTRACT(EPOCH FROM LEAST(r.end_timestamp, gen.interval_start + INTERVAL '1 hour') - GREATEST(r.start_timestamp, gen.interval_start))
+			)
+		END AS min_rate,
+
+		tsrange(gen.interval_start, gen.interval_start + INTERVAL '1 hour', '()') AS time_interval
+
+	FROM readings r
+	INNER JOIN meters m ON r.meter_id = m.id
+	INNER JOIN units u ON m.unit_id = u.id
+	CROSS JOIN LATERAL generate_series(
+		date_trunc('hour', r.start_timestamp),
+		-- Subtract 1 interval width because generate_series is end-inclusive
+		date_trunc_up('hour', r.end_timestamp) - INTERVAL '1 hour',
+		INTERVAL '1 hour'
+	) gen(interval_start)
 	GROUP BY r.meter_id, gen.interval_start, u.unit_represent
-	-- The order by ensures that the materialized view will be clustered in this way.
-	ORDER BY gen.interval_start, r.meter_id;
-
+)
+SELECT
+	m.id AS meter_id,
+  	SUM(bh.reading_rate * c.slope + c.intercept) AS reading_rate,
+  	SUM(bh.min_rate     * c.slope + c.intercept) AS min_rate,
+  	SUM(bh.max_rate     * c.slope + c.intercept) AS max_rate,
+  	bh.time_interval,
+  	c.destination_id AS graphic_unit_id
+
+FROM base_hourly bh
+JOIN meters m ON m.id = bh.meter_id
+JOIN units  u ON u.id = m.unit_id
+JOIN cik c ON c.source_id = m.unit_id AND tsrange(c.start_time, c.end_time, '()') && bh.time_interval
+GROUP BY m.id, graphic_unit_id, bh.time_interval
+ORDER BY meter_id;
```

Comment:
I noticed the order by was changed and the old version commented on the clustering. Was this deliberate?